packages/address/src.ts/index.ts

```diff
@@ -18,6 +18,16 @@ import {version} from "./_version";
 
 const logger = new Logger(version);
 
+export function getAccountFromTransactionId(transactionId: string): string {
+    // TransactionId looks like this: '0.0.99999999-9999999999-999999999'
+    if (!transactionId.match(/^\d+?\.\d+?\.\d+-\d+-\d+$/)) {
```

Comment:
Actually, hedera APIs support 2 types of transactionIDs:
- `accountID@txSeconds-txNanoseconds`
- `accountID-txSeconds-txNanoseconds`

However, I'm not sure if we have to validate for both of them.

Reply:
I added support for the other format as well